expcore/Mini_games.lua

```diff
@@ -100,73 +108,117 @@ function Mini_games.get_running_game()
+            end
+        end
 
-    if mini_game.map == nil then
-        error("No map set")
-    end
+        if started_game[1] == name then
+            return "This game is already running"
+        end
 
-    if started_game[1] then
-        Mini_games.stop_game(started_game[1])
-    end
+        if mini_game.map == nil then
+            error("No map set")
+        end
 
-    for _, player in ipairs(game.connected_players) do
-        player.teleport({mini_game.positon.x,mini_game.positon.y},mini_game.map)
-    end
+        if started_game[1] then
+            Mini_games.stop_game(started_game[1])
+        end
 
-    started_game[1] = name
+        for _, player in ipairs(game.connected_players) do
+            player.teleport({mini_game.positon.x,mini_game.positon.y},mini_game.map)
+        end
 
-    for i,value  in ipairs(mini_game.events) do
-        local handler = value[1]
-        local event_name = value[2]
-        Event.add_removable(event_name,handler)
-    end
+        started_game[1] = name
 
-    for i,value  in ipairs(mini_game.onth_tick) do
-        local tick = value[1]
-        local token = value[2]
-        Event.add_removable_nth_tick(tick, token)
-    end
+        for i,value  in ipairs(mini_game.events) do
+            local handler = value[1]
+            local event_name = value[2]
+            Event.add_removable(event_name,handler)
+        end
 
-    if mini_game.commands then
-        for i,command_name  in ipairs(mini_game.commands) do
-            Commands.enable(command_name)
+        for i,value  in ipairs(mini_game.onth_tick) do
+            local tick = value[1]
+            local token = value[2]
+            Event.add_removable_nth_tick(tick, token)
         end
-    end
 
-    local start_func = mini_game.start_function
-    if start_func then
-        if parse_args then
-            local success, err = pcall(start_func,parse_args)
-            internal_error(success,err)
-        else
-            local success, err = pcall(start_func)
-            internal_error(success,err)
+        if mini_game.commands then
+            for i,command_name  in ipairs(mini_game.commands) do
+                Commands.enable(command_name)
+            end
+        end
+
+        local start_func = mini_game.start_function
+        if start_func then
+            if parse_args then
+                local success, err = pcall(start_func,parse_args)
+                internal_error(success,err)
+            else
+                local success, err = pcall(start_func)
+                internal_error(success,err)
+            end
         end
     end
 end
 
+function Mini_games.update_airtable(args)
+    local data = {}
+    data.type = "end_game"
+    data.Gold = args[1]
+    data.Gold_data = args[2]
+    data.Silver = args[3]
+    data.Silver_data = args[4]
+    data.Bronze = args[5]
+    data.Bronze_data = args[6]
+    data.server = vars.server_adress
+    return game.table_to_json(data)
+end
```

Comment:
This can only be a single assignment, also is this meant to write to file?

Reply:
nope it return as the stop func shood return the json and then its written to file

Reply:
In that case it would be better to call this function `format_airtable` since he converts your args into the format expected by airtable.

Reply:
yeah